tests/roles/backend_services/templates/openstack_version.j2

```diff
@@ -0,0 +1,134 @@
+    manilaShareImage: {{ container_registry }}/{{ container_namespace }}/openstack-manila-share:{{ container_tag }}
+    mariadbImage: {{ container_registry }}/{{ container_namespace }}/openstack-mariadb:{{ container_tag }}
+    neutronAPIImage: {{ container_registry }}/{{ container_namespace }}/openstack-neutron-server:{{ container_tag }}
+    novaAPIImage: {{ container_registry }}/{{ container_namespace }}/openstack-nova-api:{{ container_tag }}
+    novaComputeImage: {{ container_registry }}/{{ container_namespace }}/openstack-nova-compute:{{ container_tag }}
+    novaConductorImage: {{ container_registry }}/{{ container_namespace }}/openstack-nova-conductor:{{ container_tag }}
+    novaNovncImage: {{ container_registry }}/{{ container_namespace }}/openstack-nova-novncproxy:{{ container_tag }}
+    novaSchedulerImage: {{ container_registry }}/{{ container_namespace }}/openstack-nova-scheduler:{{ container_tag }}
+    octaviaAPIImage: {{ container_registry }}/{{ container_namespace }}/openstack-octavia-api:{{ container_tag }}
+    octaviaHealthmanagerImage: {{ container_registry }}/{{ container_namespace }}/openstack-octavia-health-manager:{{ container_tag }}
+    octaviaHousekeepingImage: {{ container_registry }}/{{ container_namespace }}/openstack-octavia-housekeeping:{{ container_tag }}
+    octaviaWorkerImage: {{ container_registry }}/{{ container_namespace }}/openstack-octavia-worker:{{ container_tag }}
+    openstackClientImage: {{ container_registry }}/{{ container_namespace }}/openstack-openstackclient:{{ container_tag }}
+    osContainerImage: {{ container_registry }}/{{ container_namespace }}/edpm-hardened-uefi:{{ container_tag }}
+    ovnControllerImage: {{ container_registry }}/{{ container_namespace }}/openstack-ovn-controller:{{ container_tag }}
+    ovnControllerOvsImage: {{ container_registry }}/{{ container_namespace }}/openstack-ovn-base:{{ container_tag }}
+    ovnNbDbclusterImage: {{ container_registry }}/{{ container_namespace }}/openstack-ovn-nb-db-server:{{ container_tag }}
+    ovnNorthdImage: {{ container_registry }}/{{ container_namespace }}/openstack-ovn-northd:{{ container_tag }}
+    ovnSbDbclusterImage: {{ container_registry }}/{{ container_namespace }}/openstack-ovn-sb-db-server:{{ container_tag }}
+    placementAPIImage: {{ container_registry }}/{{ container_namespace }}/openstack-placement-api:{{ container_tag }}
+    rabbitmqImage: {{ container_registry }}/{{ container_namespace }}/openstack-rabbitmq:{{ container_tag }}
+    swiftAccountImage: {{ container_registry }}/{{ container_namespace }}/openstack-swift-account:{{ container_tag }}
+    swiftContainerImage: {{ container_registry }}/{{ container_namespace }}/openstack-swift-container:{{ container_tag }}
+    swiftObjectImage: {{ container_registry }}/{{ container_namespace }}/openstack-swift-object:{{ container_tag }}
+    swiftProxyImage: {{ container_registry }}/{{ container_namespace }}/openstack-swift-proxy-server:{{ container_tag }}
+containerImages:
```

Comment:
What's the purpose of this second list? I don't see it mentioned in the docs @ https://github.com/openstack-k8s-operators/dev-docs/blob/31f8c6a347ccf83316b4ed41bd985d0817cb349d/version_updates.md.

Reply:
yes i also wasn't sure about that... i picked it up from the openstackversion in the job logs there https://logserver.rdoproject.org/openstack-periodic-container-antelope-centos9/review.rdoproject.org/rdo-jobs/master/periodic-data-plane-adoption-osp-17-to-extracted-crc/85b988c/controller/ci-framework-data/logs/openstack-k8s-operators-openstack-must-gather/namespaces/openstack/crs/openstackversions.core.openstack.org/openstack.yaml

but looking at it again with fresh eyes i think it is not needed for the customcontainer images ... the indent is wrong

will remove

Reply:
hmmm... not sure, i think i will try it on the env as well and see

for example, the cinderVolumeImages under container Images has the 'correct' format according to dev docs https://github.com/openstack-k8s-operators/dev-docs/blob/main/version_updates.md

i.e.

```
  cinderVolumeImages:
    default: {{ container_registry }}/{{ container_namespace }}/openstack-cinder-volume:{{ container_tag }}
```

it seems cinder and manila share are the exception for some reason